state/modeluser.go

```diff
@@ -170,6 +171,147 @@ func (st *State) removeModelUser(user names.UserTag) error {
 	return nil
 }
 
+// isUserSuperuser if this user has the Superuser access on the controller.
+func (st *State) isUserSuperuser(user names.UserTag) (bool, error) {
+	access, err := st.UserAccess(user, st.controllerTag)
+	if err != nil {
+		// We don't suppress NotFound becaus
```

Comment:
Please continue :)

Reply:
actually, the change to --all probably means we have introduced a bug.
Namely, you can now ask for st.ModelSummariesForUser for users that don't exist at all.
Maybe that's not a problem, but it may be better to give a "no such user" rather than an empty list of models.
Thoughts?
I added a TODO so that we can decide how we want this to behave.